go/store/nbs/store.go

```diff
@@ -214,7 +214,10 @@ func (nbs *NomsBlockStore) UpdateManifest(ctx context.Context, updates map[hash.
 		return
 	}
 
-	nbs.checkAllManifestUpdatesExist(ctx, updates)
+	err = nbs.checkAllManifestUpdatesExist(ctx, updates)
+	if err != nil {
+		return
+	}
```

Comment:
oooh! nice catch! I've been curious about swallowed errors like this that a linter could find in our code base. I'm gunna play with [errcheck](https://github.com/kisielk/errcheck) and see what it can find.